src/pages/signin/SAMLSignInPage/index.native.tsx

```diff
@@ -17,15 +19,33 @@ import ROUTES from '@src/ROUTES';
 function SAMLSignInPage() {
     const [account] = useOnyx(ONYXKEYS.ACCOUNT);
     const [credentials] = useOnyx(ONYXKEYS.CREDENTIALS);
-    const samlLoginURL = `${CONFIG.EXPENSIFY.SAML_URL}?email=${credentials?.login}&referer=${CONFIG.EXPENSIFY.EXPENSIFY_CASH_REFERER}&platform=${getPlatform()}`;
     const [showNavigation, shouldShowNavigation] = useState(true);
+    const [SAMLUrl, setSAMLUrl] = useState('');
+    const webViewRef = useRef<WebView>(null);
+
+    useEffect(() => {
+        // If we've already gotten a url back to log into the user's IdP, then don't re-fetch it
+        if (SAMLUrl) {
+            return;
+        }
+
+        const body = new FormData();
+        body.append('email', credentials?.login ?? '');
```

Comment:
What happens if email is set as an empty string?

Reply:
I think rn the iDP page fails to load so they get stuck on the loading screen, i'll update the logic to return early if there's no login to pass.